### DISCUSSION TITLE
Linking auth.users to another table with a foreign key?

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#450] [Questions]
Is there a way that I can add the UUID of the auth.users table as the foreign key of another table? It is not available as an option when selecting a table to add as a foreign relation.

**Comment by silentworks** [comment]
Do you mean from the UI when creating an entry? if yes this doesn't seem possible at the moment so I just copied the UUID from the auth.users table and paste it in manually. 

If you mean when creating the schema then I don't think this is possible as the `auth.users` table is not a public table, the docs recommend that you create a `public.users` table that will have the same UUID as the `auth.users` table. You can use a trigger to make sure that the UUID is added to the `public.users` table when a new `auth.users` is created.

You can find more about this here https://supabase.io/docs/guides/auth#create-a-publicusers-table

**Reply by kiwicopple** [reply]
Thanks for chipping in @silentworks ! This is correct. For security reasons we don't expose the `auth` schema to the API.
